src/autosuggest/internal.tsx

```diff
@@ -188,8 +186,8 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
     ...props,
     isEmpty,
     isFiltered,
-    recoveryText,
-    errorIconAriaLabel,
+    getRecoveryText: () => i18n('errorIconAriaLabel', restProps.errorIconAriaLabel),
+    getErrorIconAriaLabel: () => i18n('recoveryText', restProps.recoveryText),
```

Comment:
The property names seem to have gotten mixed up here
```suggestion
    getRecoveryText: () => i18n('recoveryText', restProps.recoveryText),
    getErrorIconAriaLabel: () => i18n('errorIconAriaLabel', restProps.errorIconAriaLabel),
```